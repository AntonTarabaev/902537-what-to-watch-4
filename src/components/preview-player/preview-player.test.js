import PreviewPlayer from "@components/preview-player/preview-player";

it(`Should PreviewPlayer render correctly`, () => {
  const tree = renderer.create(
      <PreviewPlayer>
        <video/>
      </PreviewPlayer>, {
        createNodeMock() {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
