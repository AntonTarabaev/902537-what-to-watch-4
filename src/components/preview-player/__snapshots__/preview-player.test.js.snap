// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmCard render correctly 1`] = `
<video
  height="175"
  width="280"
/>
`;
